### ISSUE TITLE
Do you have any plans to support OpenFeature?

### ISSUE THREAD

**Issue by cosmic-flood** [uber/piranha#638]
Hi, 

Piranha is great for removing obsolete flags. Today, more developers started to implement feature flags with OpenFeature standard, do you have any plans to support OpenFeature?

**Comment by thomaspoignant** [comment]
I agree that it would be awesome to have a version working with OpenFeature.
I am part of the technical community, if you need some input from us feel free to reach us.
